src/components/mapPageComponents/logic/predict.ts

```diff
@@ -0,0 +1,46 @@
+import { BOSTON_BOUNDS } from '../mapView';
+
+// Map logic for making predictions from user input
```

Comment:
Is this logic all found within the Google API?

Reply:
There's documentation for `getPlacePredictions` and `getDetails` and I figured out how to use them together for automatically entering the first prediction in input (not sure if that's the question).